synapse/common.py

```diff
@@ -850,3 +850,8 @@ async def genrnext(g):
         yield nextvalu
 
         curvs[nextindx] = await genrnext(genrs[nextindx])
+
+def iterTagDown(fqdn):
```

Comment:
This looks like its a clone of ``iterFqdnUp`` from certdir?

Reply:
It is, but the intent behind each use case is so different that it felt wrong to call `iterFqdnUp` when what i was actually doing was `iterTagDown` .  We could utilitize into something like `iterDotsLeftToRight` or something i guess?

Reply:
Just the function name and argument name present a bit of a "what is this for?" question.

Reply:
haha, yea.  def should rename the `fqdn` arguement to `tag`  :D